app/controllers/papers_controller.rb

```diff
@@ -16,6 +16,7 @@ def show
   # GET /papers/new
   def new
     @paper = Paper.new
+    @authors = 5.times {@paper.authors.build}
```

Comment:
gotcha.  I'll just put 5 where the variable is sitting in the form then.
I'll see if I run into any problems.

On Wed, Jul 20, 2016 at 4:25 PM, Brian Kung <notifications@github.com>
wrote:

> In app/controllers/papers_controller.rb
> <https://github.com/briankung/projectcredo/pull/27#discussion_r71609814>:
>
> > @@ -16,6 +16,7 @@ def show
> >    # GET /papers/new
> >    def new
> >      @paper = Paper.new
> > +    @authors = 5.times {@paper.authors.build}
>
> So @authors = 5.times { @paper.authors.build } stores 5 into the @authors
> instance variable
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/27/files/1229611407cf11395d83618e9ba75eff1c86c772#r71609814>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACKsZJsYnhgM7X2fE2UdSoXZ2Sk40u7Gks5qXpJegaJpZM4JRHwf>
> .
>